### DISCUSSION TITLE
create rfc over urlon and send rfc urlon to IETF

### DISCUSSION THREAD

**Discussion by ghost** [ietf-tools/datatracker#4473] [Help / Questions] [answered]
Hi everyone. 

I'm thinking of creating an RFC about URLON(URL Object Notation) which is a concept of url in JSON object... and originally designed by [vjeux](https://blog.vjeux.com/2011/javascript/urlon-url-object-notation.html). There is no RFC available today about URLON, although there are several implementations about urlon as: [cerebral/url](https://github.com/cerebral/urlon) or [Double-oxygeN/urlon-nim](https://github.com/Double-oxygeN/urlon-nim). In this sense, I would like to know where I can find the rfc file template so that I can create this rfc over urlon and send it to the IETF. 

My questions are these:
1. where can I find the RFC file template for IETF?
2. where I send the RFC file to the IETF?
3. have you all heard about the urlon(url object notation) implementation?

**Comment by rjsparks** [comment, marked as answer]
Welcome to the IETF. As you get further along, you'll find there were better places for these questions. Please help us, when you get there, by suggesting changes to www.ietf.org (or other properties) that would have made finding the answers to your question easier.

https://authors.ietf.org/en/choosing-a-format-and-tools will help you with how to build a draft. That wiki also contains an overview of the entire process - be sure to explore the menu on the left.

https://www.ietf.org/how/ids/ should also be useful (menu->participate->Internet Drafts).

Discussing new ideas is best done on the email lists for relevant groups. Given the topic, the DISPATCH list would be a good place to start: https://mailarchive.ietf.org/arch/browse/dispatch/ (see also https://datatracker.ietf.org/group/dispatch/about/).

**Reply by larseggert** [reply]
What @rjsparks said. Also, familiarize yourself with https://www.ietf.org/about/note-well/ first, especially BCP 78 and 79 (linked from there.) Anything you email to the IETF or submit as a document counts as a contribution.
